src/types.ts

```diff
@@ -1407,9 +1424,25 @@ export type UserSort<UserType = UnknownType> =
   | Sort<UserResponse<UserType>>
   | Array<Sort<UserResponse<UserType>>>;
 
-export type QuerySort<ChannelType = UnknownType, UserType = UnknownType> =
+export type SearchRelevanceSort = { relevance?: AscDesc };
+
+export type SearchMessageSortBase<MessageType = UnknownType> =
+  | SearchRelevanceSort
+  | Sort<MessageType>
+  | { [field: string]: AscDesc };
```

Comment:
Is there not a tighter set of fields here? `Sort<MessageType>` covers all user created ones but would `Message` or `MessageBase` with `Sort` be usable for the rest instead of [field: string] ?